test/powershell/Language/Scripting/MacLineEndings.Tests.ps1

```diff
@@ -0,0 +1,69 @@
+﻿Describe 'Tests for mac line endings' -Tags "CI" {
+    BeforeAll {        
+        $newline = "`n"

```

Comment:
fixed

---
In reply to: [78842007](https://github.com/Powershell/Powershell/pull/2118#discussion_r78842007) [](ancestors = 78842007)